libs/platform/src/lib/components/form/form-group/form-field/form-field.component.ts

```diff
@@ -370,4 +393,56 @@ export class FormFieldComponent implements FormField, AfterContentInit, AfterVie
             throw new Error('fdp-form-field must contain [id] binding.');
         }
     }
+
+    /** @hidden */
+    private _setColumnNumber(): void {
+        try {
+            this._xlColumnNumber = this.columnLayout['XL'];
+            this._lgColumnNumber = this.columnLayout['L'];
+            this._mdColumnNumber = this.columnLayout['M'];
+            this._sColumnNumber = this.columnLayout['S'] || 1;
```

Comment:
Maybe it's better to do like this:
```typescript
this._sColumnNumber = this.columnLayout['S'] || 1;
this._mdColumnNumber = this.columnLayout['M'] || this._sColumnNumber;
this._lgColumnNumber = this.columnLayout['L'] || this._mdColumnNumber;
this._xlColumnNumber = this.columnLayout['XL'] || this._lgColumnNumber;
```

This way we will inherit previous column layout without the need to specify all breakpoints if they are the same

Reply:
fixed